src/api/spec/controllers/webui/obs_factory/distributions_controller_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+require 'webmock/rspec'
+# WARNING: If you change owner tests make sure you uncomment this line
```

Comment:
We don't do that anymore. We do this in general in [spec/support/vcr.rb](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/support/vcr.rb#L20).